src/node_http_parser.cc

```diff
@@ -167,6 +168,23 @@ class Parser : public AsyncWrap, public StreamListener {
     num_fields_ = num_values_ = 0;
     url_.Reset();
     status_message_.Reset();
+
+    Local<Object> obj = object();
+    Local<Value> cb = obj->Get(env()->context(),
+                               kOnMessageBegin).ToLocalChecked();
+
+    if (!cb->IsFunction())
+      return 0;
+
+    Local<Value> argv[0];
+    MaybeLocal<Value> r = MakeCallback(cb.As<Function>(), 0, argv);
```

Comment:
The line above doesn’t compile on Windows – you can just use `nullptr` as the last argument here.

Reply:
I have fixed this line. Thanks!